src/_ert/forward_model_runner/client.py

```diff
@@ -82,14 +82,14 @@ def __init__(
 
         self._max_retries = max_retries
         self._timeout_multiplier = timeout_multiplier
-        self.websocket: Optional[WebSocketClientProtocol] = None
+        self.websocket: Optional[ClientConnection] = None
         self.loop = new_event_loop()
 
-    async def get_websocket(self) -> WebSocketClientProtocol:
+    async def get_websocket(self) -> ClientConnection:
         return await connect(
             self.url,
             ssl=self._ssl_context,
-            extra_headers=self._extra_headers,
+            additional_headers=self._extra_headers,
```

Comment:
Can we rename the instance variable `self._extra_headers` to `self._additional_headers`?